charts/stateless-dns/values.yaml

```diff
@@ -116,8 +116,16 @@ externalDNS:
   interval: 30s  # The amount of time between external-dns runs.
   triggerLoopOnEvent: true  # Make external-dns to act by reacting to cluster events besides the polling with the interval above.
   policy: sync  # How DNS records are synchronized between sources and providers. sync will delete entries created by external-dns, it could also destroy entries that are not owner by external-dns if at some point it mark it as owned. upsert-only will create entries but will prevent the deletion of them.
-  txtOwnerId: ""  # Each of the entris added by external-dns will create a TXT entry in the DNS too to mark external-dns as it owner. This is an ID that will be added to that TXT entry.
-  txtPrefix: ""  # Each of the entris added by external-dns will create a TXT entry in the DNS too to mark external-dns as it owner. The TXT entry name will be the entry that is being created plus this prefix.
+  txtOwnerId: ""  # Each of the entries added by external-dns will create a TXT entry in the DNS too to mark external-dns as it owner. This is an ID that will be added to that TXT entry.
+  # Each of the entries added by external-dns will create a TXT entry in the DNS too to mark external-dns as it owner. The TXT entry name will be the entry that is being created plus this prefix.
+  # external-dns needs to add to the TXT registry the record type (A, NS, etc.) of the original record in order to have
+  # separate TXTs for different record types with the same name. If the special token `%{record_type}` is not specified
+  # in this prefix, external-dns will add it automatically to the domain name, e.g. `test-ns.local` for `test.local NS`.
+  # If this happens, PowerDNS would reject this record as it most likely will not have a zone for `test-ns.local`.
+  # For stateless-dns to work, this prefix must:
+  # 1. End with a dot (`.`), so all TXT records belong to the top level zone.
+  # 2. Contain the `%{record_type}` token before that dot.
```

Comment:
This comment is not entirely correct. It _is_ possible to have a different record format, but it must belong to a zone that PDNS knows.
Currently this chart is only creating zones for domains themselves, but it could be possible to create meta-zones holding only txt registries.

Reply:
Would than make sense to do it/test it now? We would have clean domains published with no useless TXT that have only meaning for the controller itself.

I am also thinking also in having a 

That way we do not leak the owner to the public.

Reply:
We can leave this in a different PR.